app/views/shared/menu/_mobile_menu.html.haml

```diff
@@ -6,7 +6,7 @@
 
     %section.left
       .ofn-logo
-        %a{href: main_app.root_path}
+        %a{href: main_logo_link(@white_label_distributor)}
```

Comment:
I think that it's a bit confusing to name the enterprise `@white_label_distributor` because it may not use white-labelling. Why can't we just use `current_distributor`? Or use that helper in the enterprise helper?

Reply:
I one case we could not use the `current_distributor`: when seeing a past order. 
That's why we do :
```
    before_action only: [:show, :edit] do
      hide_ofn_navigation(@order.distributor)
    end
```
https://github.com/openfoodfoundation/openfoodnetwork/blob/729230799d8697ed8fa1961b95e5932a60d9476f/app/controllers/spree/orders_controller.rb#L27-L30